config.go

```diff
@@ -0,0 +1,5 @@
+package tigerblood
+
+type Config struct {
+	Credentials map[string]string
```

Comment:
Using a map keeps things simpler on hawk.go. Otherwise we'd have to iterate over all credentials to find the ones corresponding to the user. I don't think there's much of a difference either way, and this way means slightly more code, so I prefer this.